src/surfaces/actionBar/actionBar.jsx

```diff
@@ -71,108 +57,124 @@ const propTypes = {
 
     /**
      * Forwards
      * Step 1: Fade out title and non-selected parent options
      */
-    const forwardsStep1Duration = `${standardFifthTiming}ms`;
+    const forwardsStep1Duration = standardFifthTiming;
 
     /**
      * Step 2: Reveal selected parent option by highlighting it.
      */
-    const forwardsStep2Delay = `${forwardsStep1Duration + 50}ms`;
-    const forwardsStep2Duration = `${standardThirdTiming}ms`;
+    const forwardsStep2Delay = forwardsStep1Duration + 50;
+    const forwardsStep2Duration = standardThirdTiming;
 
     /**
      * Step 3: Move selected parent option to top of drawer and show --sub_options
      */
-    const forwardsStep3Delay = `${forwardsStep1Duration + forwardsStep2Delay + forwardsStep2Duration}ms`;
-    const forwardsStep3Duration = `${standardThirdTiming}ms`;
+    const forwardsStep3Delay = forwardsStep1Duration + forwardsStep2Delay + forwardsStep2Duration;
+    const forwardsStep3Duration = standardThirdTiming;
 
     /**
      * Step 4: By staggering, fade in and move down sub options.
      */
-    const forwardsStep4Delay = `${forwardsStep3Delay + forwardsStep3Duration}ms`;
-    const forwardsStep4Duration = `${standardThirdTiming}ms`;
+    const forwardsStep4Delay = forwardsStep3Delay + forwardsStep3Duration;
+    const forwardsStep4Duration = standardThirdTiming;
 
     /**
      * Backwards
      * Step 1: These will happen right away, in otherwords, no delay.
      *         Move sub options up and fade out
      *         Move selected option back down
      */
-    const backwardsStep1Duration = `${standardFifthTiming}ms`;
+    const backwardsStep1Duration = standardFifthTiming;
 
     /**
      * Step 2: Move selected parent option back down and hide --sub_options
      */
-    const backwardsStep2Delay = `${backwardsStep1Duration + 50}ms`;
-    const backwardsStep2Duration = `${standardThirdTiming}ms`;
+    const backwardsStep2Delay = backwardsStep1Duration + 50;
+    const backwardsStep2Duration = standardThirdTiming;
 
     /**
      * Step 3: Remove highlight on selected option
      */
-    const backwardsStep3Delay = `${backwardsStep2Delay + backwardsStep2Duration}ms`;
-    const backwardsStep3Duration = `${standardThirdTiming}ms`;
+    const backwardsStep3Delay = backwardsStep2Delay + backwardsStep2Duration;
+    const backwardsStep3Duration = standardThirdTiming;
 
     /**
      * Step 4: Fade in drawer title
      *         By staggering, fade in and move down options.
      */
-    const backwardsStep4Delay = `${backwardsStep3Delay + backwardsStep3Duration}ms`;
-    const backwardsStep4Duration = `${standardFifthTiming}ms`;
+    const backwardsStep4Delay = backwardsStep3Delay + backwardsStep3Duration;
+    const backwardsStep4Duration = standardFifthTiming;
 
     let optionContainerStyles;
     let subOptionStyles;
 
     for (let i = 1; i <= 11; i += 1) {
+        const optionYPos = i === 1 ? 0 : actionsButtonDrawerOptionHeight * (i - 1);
```

Comment:
Is the ternary necessary here?  `i (===1) - 1 => 0 * actionsButtonDrawerOptionHeight  => 0` ... right?

Reply:
Nope it isn't. Thanks. Fixed.

Reply:
Looks simplified now.  Thanks.  Resolving this.